app/helpers/ems_cluster_helper/textual_summary.rb

```diff
@@ -254,6 +254,16 @@ def textual_aggregate_disk_capacity
     {:label => "Total Disk Capacity", :value => number_to_human_size(@record.aggregate_disk_capacity.gigabytes, :precision => 2)}
   end
 
+  def textual_cinder_disk_usage
+    return nil unless @record.kind_of?(ManageIQ::Providers::Openstack::InfraManager::EmsCluster) && @record.name.include?("BlockStorage")
```

Comment:
@rwsu I don't think so, but best to move the identification piece to one place, so we can easily modify/extend it later